js/app/dashboard/components/Task.js

```diff
@@ -18,9 +18,23 @@ const TaskCaption = ({ task }) => {
 
   const { t } = useTranslation()
 
+//  console.log( { i18n.t('ADMIN_DASHBOARD_ORDERS_ORDER', { task.metadata.order_number }  ) } )
+
+  console.log( task.metadata.order_number)
+
+  console.log( 'order_number in task:', task.metadata.order_number)
+  console.log( 'isEmpty in task:', _.isEmpty(task.metadata.order_number))
+//      { task.metadata.order_number }
+//            { if (!_.isEmpty(task.metadata.order_number)) {
+//        return { task.metadata.order_number };
+//        }
+//      return;
+//      }
+
   return (
     <span>
-      <span className="mr-1">#{ task.id }</span>
+      <span className="mr-1"> #{ task.id } | { task.metadata.order_number }
```

Comment:
When tasks haven't been generated by a marketplace order, they won't have any `order_number` metadata. 
There should be conditional code here, that checks if the `order_number` metadata exists.